tutorial/source/svi_part_iii.ipynb

```diff
@@ -343,23 +383,21 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "```\n",
     "Doing inference with use_decaying_avg_baseline=True\n",
-    "...........\n",
-    "Did 2070 steps of inference.\n",
-    "Final absolute errors for the two variational parameters (in log space) were 0.0500 & 0.0443 \n",
+    "....................\n",
+    "Did 1932 steps of inference.\n",
+    "Final absolute errors for the two variational parameters were 0.7997 & 0.0800\n",
```

Comment:
Can you confirm that this 16x increase in absolute error is ok?

Reply:
it's ok. it only appears like that because it's no longer in log space